desktop/flipper-ui-core/src/chrome/SettingsSheet.tsx

```diff
@@ -499,6 +513,46 @@ export default connect<StateFromProps, DispatchFromProps, OwnProps, Store>(
   {updateSettings, updateLauncherSettings},
 )(withTrackingScope(SettingsSheet));
 
+function AndroidUserIdField(props: {
+  defaultValue: string;
+  onChange: (path: string) => void;
+}) {
+  const activeDevice: BaseDevice = useSelector(getActiveDevice);
+  const [users, setUsers] = useState([] as {id: string; name: string}[]);
+
+  activeDevice
```

Comment:
Best run shell commands in an effect, with the current setup the command is fired on every render

Reply:
As said on comments, I don't know React. Please feel free to edit that part, as I don't have knowledge about these "useEffect" things. Just a warn: when you say "the command is fired on every render", it may be fine. Best would be to run this command each time selected device change (and user is opening settings sheet) or you could have bad users ids shown.